### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by balloob**
Add CI to verify linters, publish to PyPI, dependabot and release-drafter

### CODE REVIEW

**Review by agners — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    strategy:
+      matrix:
+        python-version:
+          - "3.9"
+          - "3.10"
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 2
```
Comment (ludeeus) [inline .github/workflows/test.yml:24]: Why is this needed here?

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
```
Comment (ludeeus) [inline .github/workflows/release-drafter.yml:12]: If comment is needed it should state "main" an not "master"
